hoot-core/src/main/cpp/hoot/core/io/OgrReader.cpp

```diff
@@ -975,6 +975,36 @@ Meters OgrReaderInternal::_parseCircularError(Tags& t)
   {
     bool ok;
     double a = t["error:circular"].toDouble(&ok);
```

Comment:
The error:circular and corresponding accuracy tags should only ever store length values, not velocities.